mobile/src/screens/JoinGame/index.js

```diff
@@ -48,13 +47,20 @@ export default function JoinGame({
 
             case GameSessionState.CHOOSE_TRICKIEST_ANSWER:
             case GameSessionState.PHASE_2_DISCUSS:
-            case GameSessionState.PHASE_2_RESULTS:
                 return navigation.navigate("PhaseTwoBasicGamePlay", {
                     gameSession,
                     team: gameSession.teams.find((team) => team.id === teamId),
                     teamMember,
                 })
 
+            case GameSessionState.PHASE_1_RESULTS:
+            case GameSessionState.PHASE_2_RESULTS:
+                return navigation.navigate("PhaseResult", {
+                    gameSession,
+                    team: gameSession.teams.find((team) => team.id === teamId),
+                    teamMember,
+                })
```

Comment:
We'd also need avatars from select team screen to be passed along to all screens

Reply:
Got it. Will do.